maven/bnd-testing-maven-plugin/src/main/java/aQute/bnd/maven/testing/plugin/TestingMojo.java

```diff
@@ -131,8 +126,10 @@ private void testing(Run run) throws Exception {
 			if (run.getErrors().isEmpty())
 				return;
 
-			System.out.println(Strings.join("\n", run.getErrors()));
-			throw new MojoExecutionException("Test had errors");
+			throw new MojoExecutionException(
+					Strings.join("\n", new Object[] {
+							"Test had errors:", Strings.join("\n", run.getErrors())
+					}));
 		} catch (Exception e) {
```

Comment:
Doesn't this catch the MojoExecutionException you just threw above? And rethrow it? That seems terrible.